rfc/rfc011-verifiable-credential.md

```diff
@@ -0,0 +1,215 @@
+# RFC011 Verifiable Credentials
+
+|  |  |
+| :--- | :--- |
+| Nuts foundation | W.M. Slakhorst |
+| Request for Comments: 011 | Nedap |
+|  | February 2021 |
+
+## RFC011 Verifiable Credentials
+### Abstract
+
+This RFC describes the generic requirements for [Verifiable Credentials (VC)](https://www.w3.org/TR/vc-data-model/) within the Nuts specification. 
+It also lists all required chapters for a specific VC RFC. This RFC uses the [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/) specification.
+
+### Status
+
+This document is currently a draft.
+
+### Copyright Notice
+![](../.gitbook/assets/license.png)
+
+This document is released under the [Attribution-ShareAlike 4.0 International \(CC BY-SA 4.0\) license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## 1. Introduction
+
+It's to be expected that multiple sources must be supported that can claim certain information about a subject. 
+These claims have to be structured in such a way, that they are verifiable and searchable. 
+Examples of claims are a company name, and it's chamber of commerce number. Users will search on the name of an organization to find out which services are supported.
+This will directly influence the interaction the user can have with the system of that organization. 
+Making sure this information is correct and that it can be trusted is therefore extremely important.
+
+## 2. Terminology
+
+* **DID**: [Decentralized Identifiers](https://www.w3.org/TR/did-core/).
+* **holder**: The party that receives a VC from an issuer.  
+* **issuer**: The party that issues and signs a VC.  
+* **proof**: Proof asserting that a VC is valid using cryptography.
+* **VC**: Verifiable Credential according to the [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/).
+* **VP**: Verifiable Presentation according to the [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/).
+
+## 3. W3C Verifiable Credential
+
+To support a variety of claims, the [W3C Verifiable Credential specification](https://www.w3.org/TR/vc-data-model/) is used. 
+Every Nuts specific VC and proof type must follow the W3C specification. 
+If the W3C specification offers options, then the specific VP must specify which option is to be used.
+All VCs MUST use DIDs as specified in [RFC004](rfc004-distributed-document-format.md) for the issuer. 
+
+### 3.1 Supported proofs
+
+The following proof types must be supported:
+
+#### 3.1.1 JsonWebSignature2020
+
+resources:
+
+* https://w3c-ccg.github.io/ld-proofs/ 
+* https://tools.ietf.org/html/rfc7797
+* https://w3c-ccg.github.io/lds-jws2020
+
+The algorithm used to create the signature is as follows:
+
+- take the input Verifiable Credential and remove the `proof` field. Store as `raw_vc`.
```

Comment:
Maybe we should mention it's just a JWS in compact serialization format with the payload removed? Because now the pseudocode looks complex, mentioning it's basically a JWS puts it into perspective.